README.md

```diff
@@ -1,2 +1,209 @@
+            "permissions": ["CallCharacter", "GetHelp", "GetCharacterList"]
+        },
+        {
+            "name": "moderator",
+            "parent": "member",
+            "permissions": []
+        },
+        {
+            "name": "administrator",
+            "parent": "moderator",
+            "permissions": []
+        },
+        {
+            "name": "Mod - La Table Ronde",
+            "parent": "moderator",
+            "permissions": []
+        },
+        {
+            "name": "Mod - Bartender",
+            "parent": "moderator",
+            "permissions": []
+        },
+        {
+            "name": "Mod - Croupier",
+            "parent": "moderator",
+            "permissions": []
+        },
+        {
+           "name": "Admin - Conseil de Guerre",
+           "parent": "administrator",
+           "permissions": []
+        },
+        {
+            "name": "Pédèstres seniors",
+            "parent": "member",
+            "permissions": []
+        },
+        {
+            "name": "Semi Croustillants",
+            "parent": "member",
+            "permissions": []
+        },
+        {
+            "name": "Indépendentistes Gallois",
+            "parent": "member",
+            "permissions": []
+        },
+        {
+            "name": "Team Chateau",
+            "parent": "member",
+            "permissions": []
+        }
+    ]
+}
+```
+
+#### Characters
+
+The Characters package bundle all the features related to the quoting of characters.
+
+The properties are:
+
+ - **folder:** The path of the folder from which the characters are generated.
+
+A character is generated from a json file and will take the name of the file. For example, arthur.json becomes arthur for the bot.
+
+Example for arthur.json
+
+```json
+{
+    "name": "arthur",
+    "sentences": [
+        "C’est vrai ce qu’on dit, vous êtes le fils d’un démon et d’une pucelle ? Vous avez plus pris de la pucelle.",
+
+        "Ça vous fait pas mal à la tête de glandouiller vingt-quatre heures sur vingt-quatre ?",
+
+        "Y a pas à dire, dès qu'il y a du dessert, le repas est tout de suite plus chaleureux !"
+    ]
+}
+```
+
+## Run
+
+### Simple
+
+
+You can run it by calling it from your shell:
+
+~~~ shell script
+kadok.exe -t <bot-token> (-p <properties-file>)
+~~~
+
+Available options:
+
+ - -t \<bot-token\> : **required**, bot's token to access discord's api
+ - -p \<properties-file\> : optionnal, bot's properties file, by default "./properties.yaml"
+
+### Using Docker
+
+> Soon to come !
+
+# Contributors
+
+  * [elBichon](https://github.com/elBichon)
+  * [Sarlam](https://github.com/sarlam)
+  * [Terag](https://github.com/Terag)
+
+# Credits
+
+The bot is developed using the SDK DiscordGo: https://github.com/bwmarrin/discordgo
+
+The bot and characters it quotes are inspired from the universe of Kaamelott by Alexandre Astier: https://kaamelott.com/
+
+# License
+
+GNU GENERAL PUBLIC LICENSE Version 3, 29 June 2007
```

Comment:
we should probably license Kaamelott as well

Reply:
A line licensing Kaamelott has been added. Waiting for confirmation